docs/quickstart/quickstart_chains/cosmos-agoric.md

```diff
@@ -0,0 +1,344 @@
+
+1. Open your browser and head to `http://localhost:3000`.
+
+2. You will see a GraphQL playground in the browser and the schemas which are ready to query.
+
+3. Find the _Docs_ tab on the right side of the playground which should open a documentation drawer. This documentation is automatically generated and it helps you find what entities and methods you can query.
+
+Try the following queries to understand how it works for your new SubQuery starter project. Don’t forget to learn more about the [GraphQL Query language](../../run_publish/query.md).
+
+```graphql
+{
+  query {
+    transferEvents(first: 2) {
+      nodes {
+        id
+        blockHeight
+        txHash
+        recipient
+        sender
+        amount
+      }
+    }
+    messages(first: 2) {
+      nodes {
+        id
+        blockHeight
+        txHash
+        from
+        to
+        amount
+      }
+    }
+  }
+}
+```
+
+You will see the result similar to below:
+
+```json
+{
+  "data": {
+    "query": {
+      "transferEvents": {
+        "nodes": [
+          {
+            "id": "69D296C6E643621429959A5B25D2F3DE1F1A67A5481FC1B7986322DBEA61BF8D-0-217",
+            "blockHeight": "12310659",
+            "txHash": "69D296C6E643621429959A5B25D2F3DE1F1A67A5481FC1B7986322DBEA61BF8D",
+            "recipient": "agoric14nnv35u887jjf2sy24cfvmh2dq3h58c7lv5sxk",
+            "sender": "agoric1xa6pnqj4jpw06n3ffdrtgc5644rgd79hfzq7mc",
+            "amount": "69979701ubld"
+          },
+          {
+            "id": "B35A81D98AAE07FB638064FD1CA2F06D72CF877B18F55A285BC162F321A2FF16-0-218",
+            "blockHeight": "12308513",
+            "txHash": "B35A81D98AAE07FB638064FD1CA2F06D72CF877B18F55A285BC162F321A2FF16",
+            "recipient": "agoric153xw08gz7emm4y28tekzgy3ewmgdykdz6csznj",
+            "sender": "agoric142sgu52x5d9w79yzp5c43z0z5n4dstvxp7lajw",
+            "amount": "974720000ubld"
+          }
+        ]
+      },
+      "messages": {
+        "nodes": [
+          {
+            "id": "B35A81D98AAE07FB638064FD1CA2F06D72CF877B18F55A285BC162F321A2FF16-0",
+            "blockHeight": "12308513",
+            "txHash": "B35A81D98AAE07FB638064FD1CA2F06D72CF877B18F55A285BC162F321A2FF16",
+            "from": "agoric142sgu52x5d9w79yzp5c43z0z5n4dstvxp7lajw",
+            "to": "agoric153xw08gz7emm4y28tekzgy3ewmgdykdz6csznj",
+            "amount": "[{\"denom\":\"ubld\",\"amount\":\"974720000\"}]"
+          },
+          {
+            "id": "627DFE0E4C1188D66FCE63D37ABE40907F9157F5FEC6BB7C98888629E12AFDAE-0",
+            "blockHeight": "12309508",
+            "txHash": "627DFE0E4C1188D66FCE63D37ABE40907F9157F5FEC6BB7C98888629E12AFDAE",
+            "from": "agoric142sgu52x5d9w79yzp5c43z0z5n4dstvxp7lajw",
+            "to": "agoric1nrtnwnu8930dm8sjlghpjxzykl4jy0dazrlxxc",
+            "amount": "[{\"denom\":\"ubld\",\"amount\":\"1178914251\"}]"
+          }
+        ]
+      }
+    }
+  }
+}
+```
```

Comment:
Can you please add a link to see this project code here as well

Reply:
Done.